dlp/dlp_inspect_image_file.rb

```diff
@@ -0,0 +1,60 @@
+# Copyright 2018 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
Newline in-between copyright and code.

Reply:
Fixed https://github.com/GoogleCloudPlatform/ruby-docs-samples/pull/356/commits/524649b022c4d1f2ca6ba12aaeac5f6e748c9c86